enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -588,6 +621,12 @@ func (c *FirecrackerContainer) LoadSnapshot(ctx context.Context, workspaceDirOve
 			ExecFile:       firecrackerBinPath,
 			ChrootStrategy: fcclient.NewNaiveChrootStrategy(""),
 		},
+		Snapshot: fcclient.SnapshotConfig{
+			MemFilePath:         fullMemSnapshotName,
+			SnapshotPath:        vmStateSnapshotName,
+			EnableDiffSnapshots: true,
+			ResumeVM:            false,
```

Comment:
Without knowing what this does -- are we still able to suspend a vm, snapshot it, reload the snapshot into a new VM and then resume it?

Reply:
Manually tested it; works 👍